apps/test-app-runtime/package.json

```diff
@@ -30,7 +30,7 @@
     "jest": "^29",
     "ts-jest": "^29",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5"
+    "typescript": "^5.0.4"
```

Comment:
stick with the convention of `^5`?

Reply:
I didn't think it mattered for the test packages.